pages/components/TextInput.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Form from "react-bootstrap/Form";
+import Col from "react-bootstrap/Col";
+
+const TextInput = ({ name, label, stream, handleChange }) => {
+  return (
+    <Form.Group
+      as={Col}
+      sm={12}
+      lg={name.includes("audio") || name === "channels" ? 6 : 12}
```

Comment:
this includes generates an error already in the build process
<img width="1168" alt="Skärmavbild 2020-03-12 kl  15 17 07" src="https://user-images.githubusercontent.com/624182/76530965-c4440f00-6474-11ea-8005-719340cac3bb.png">